core/cmd/remote_client.go

```diff
@@ -623,7 +647,12 @@ func parseResponse(resp *http.Response) ([]byte, error) {
 	if resp.StatusCode == http.StatusUnauthorized {
 		return b, errUnauthorized
 	} else if resp.StatusCode >= http.StatusBadRequest {
-		return b, errors.New("Error")
+		errorMessage, err := parseErrorResponseBody(b)
```

Comment:
Is it possible that `b` can be empty in some cases? Will your `json.Unmarshal` fail in this case?

Reply:
Added check if `b` is empty